datumaro/util/image.py

```diff
@@ -78,18 +81,15 @@ def save_image(path, image, create_dir=False, dtype=np.uint8, **kwargs):
     if dtype == np.int32:
         backend = _IMAGE_BACKENDS.PIL
     if backend == _IMAGE_BACKENDS.cv2:
-        import cv2
+        # cv2.imwrite does not support paths that are not representable
+        # in the locale encoding on Windows, so we write the image bytes
+        # ourselves.
 
-        params = []
+        ext = osp.splitext(path)[1]
+        image_bytes = encode_image(image, ext, dtype=dtype, **kwargs)
 
-        ext = path[-4:]
-        if ext.upper() == '.JPG':
-            params = [
-                int(cv2.IMWRITE_JPEG_QUALITY), kwargs.get('jpeg_quality', 75)
-            ]
```

Comment:
Why this code was removed? OpenCV and PIL have different jpeg quality defaults.

Reply:
This code is reused through `encode_image`.